products/casp/main.pm

```diff
@@ -32,26 +32,42 @@ my $distri = testapi::get_var("CASEDIR") . '/lib/susedistribution.pm';
 require $distri;
```

Comment:
but necessary to initialize the distribution object for os-autoinst. Yeah, it looks a bit weird. If you have an idea how to improve, go ahead :-)